File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/entity/SysDictItem.java
package ink.ckx.mo.system.api.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import ink.ckx.mo.common.core.base.BaseEntity;
import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class SysDictItem extends BaseEntity {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String typeCode;

    private String name;

    private String value;

    private StatusEnum status;

    private Integer sort;

    private Boolean defaulted;

    private String remark;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/entity/SysDictItem.kt
package ink.ckx.mo.admin.api.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import ink.ckx.mo.common.core.base.BaseEntity
import ink.ckx.mo.common.web.enums.StatusEnum

data class SysDictItem(
    @TableId(type = IdType.ASSIGN_ID)
    var id: Long? = null,
    var typeCode: String? = null,
    var name: String? = null,
    var value: String? = null,
    var status: StatusEnum? = null,
    var sort: Int? = null,
    var defaulted: Boolean? = null,
    var remark: String? = null,
) : BaseEntity() 